RetailCoder.VBE/UI/Inspections/InspectionResultsControl.xaml

```diff
@@ -19,8 +17,6 @@
                 <ResourceDictionary Source="../Controls/ToolBar.xaml"/>
             </ResourceDictionary.MergedDictionaries>
 
-            <converters:InvertBoolValueConverter x:Key="InvertBoolValue" />
```

Comment:
What's up with the deletions in this file?

Reply:
That came up as an unused converter.

Reply:
@IvenBach are you 100% sure that no XAML is referring to it? XAML markup escapes ReSharper static code analysis, and a missing class will not break the build either - but things *will* go BOOM at run-time.

Reply:
fulltext search in the file returns no results, I'm inclined to agree that this one is unused